src/content/docs/CDN/CDN-case-study.md

```diff
@@ -0,0 +1,131 @@
+
+## Application example
+
+This is a step by step guide on how to use bootstrap CSS with CDN on a new HTML document
+
+1. First start by creatig a new HTML document, Bootstrap requires HTML5 doctype.
+2. in the `<head>` tag of the HTML file add a `<link>` tag, this link tag is what is going to import Bootsrap using a CDN. for this example JSdelivr is used copy and paste this link and place it in the `<head>` of the HTML document:
+
+```html
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+  integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+  crossorigin="anonymous"
+/>
+```
+
+3. The HTML page should look something like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bootstrap CDN example</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body></body>
+</html>
+```
+
+4. Bootstrap CSS should now be implemented to test if Bootstrap CSS is implemented correctly copy and paste this code in the `<body>` tag which is shown below:
+
+```html
+<body><h1 class="bg-primary">My First Heading</h1></body
+```
+
+now open a live server and look at the Heading, if the background is blue Bootstrap was successfully implemented if the background of the Heading is not blue Bootstrap CSS was not implemented correctly.
```

Comment:
Fix the tone here
```suggestion
5. Now, open with live server and observe the Heading. If the background is blue, Bootstrap has been successfully implemented. If the background of the Heading is not blue, Bootstrap CSS has not been implemented correctly.
```

Reply:
Thank you very much for the sugestions, I have changed the text to the suggested grammar and tone